src/router/storeSync/CrossHairParamConfig.class.js

```diff
@@ -99,6 +77,21 @@ function generateCrossHairUrlParamFromStoreValues(store) {
     return null
 }
 
+function acceptedValues(store, query) {
+    if (query) {
+        const parts = query.split(',')
+        let crossHair = parts[0]
+        let crossHairPosition = [parseFloat(parts[1]), parseFloat(parts[2])]
+
+        return (
+            (typeof query === 'string' || query instanceof String) &&
```

Comment:
Testing for string after the split don't make sense, moreover the input should come from the vue router which is either a string or null/undefined, meaning that the if at line 81 would test for existence and non empty string.